### PR TITLE
Makefile and launch files

### PR DESCRIPTION
**Pull request by kylestach**
## Description
Makefile and launch support for ROS.

This adds sim2play launch file, in which two ROS graphs are instanced under the `/yellow` and `/blue` namespaces.

## Associated Issue
Issue #1521

## Design
New ROS graph:
image

### Standard sim launch

Run `make run-sim`

Expected result: a soccer window should appear. Soccer should work in simulation.

### 2-player sim launch

Run `make run-sim2play`

Expected result: two soccer windows should show up. Closing either should kill off all programs.

### Other targets
 - `make`: builds code
 - `make run-headmore`: runs sim with grSim visible
 - `make run`: runs soccer only, without simulator (in non-sim mode)

You can also use `make` followed by `ros2 launch` directly.

### CODE REVIEW

**Review by liquidmetal9015 — COMMENTED** [review]
run-sim, headmore, and run are working fine for me. I'm currently having some issues with sim2play but I suspect I might be doing something wrong sourcing wise or otherwise.

Currently Sim2Play is throwing this on startup.
[soccer-3] [INFO] [1603139153.620729864] [yellow.soccer_config_client]: [ConfigClient] Connected!
[soccer-3] terminate called after throwing an instance of 'boost::wrapexcept<boost::system::system_error>'
[soccer-3]   what():  bind: Address already in use

I would also ask does make clean work properly here?

Thanks making these targets.

**Comment by kylestach** [comment]
I'm going to leave some of the additional startup race condition issues to a later PR, I think. For now this should be good.

**Review by liquidmetal9015 — APPROVED** [review]
Looks good to me

**Review by HussainGynai — APPROVED** [review]
LGTM

File: rj_config/src/config_server/main.cpp [hunk 1]
```diff
@@ -1,12 +1,53 @@
+#include <fmt/format.h>
 #include <rclcpp/rclcpp.hpp>
 
 #include <config_server/config_server.hpp>
 
 using config_server::ConfigServer;
+using config_server::GameSettingsMsg;
+
+/**
+ * Parse game settings from the arguments. TODO: Unify this with the parsing in main().
```
Comment (oswinso) [inline rj_config/src/config_server/main.cpp]: Issue number

**Review by oswinso — DISMISSED** [review]
Just the one TODO issue number.

**Pull request merged**